spec/requests/application_controller_spec.rb

```diff
@@ -143,6 +143,22 @@
         expect(response.status).to eq(404)
         expect(response.body).to_not include('google.com/search')
       end
+
+      it 'should cache results' do
+        $redis.flushall
+
+        topic1 = Fabricate(:topic)
+        get '/t/nope-nope/99999999'
+        expect(response.status).to eq(404)
+        expect(response.body).to include(topic1.title)
+
+        topic2 = Fabricate(:topic)
+        get '/t/nope-nope/99999999'
+        expect(response.status).to eq(404)
+        expect(response.body).to_not include(topic2.title)
```

Comment:
oops sorry I think we need to assert that the body contains `topic1` but not `topic2`.